tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -53,6 +53,9 @@ On the right pane of the endpoint details page, you can click the **Properties**
 - **Request Method**: the HTTP method of the endpoint. The following methods are supported:
 
     - `GET`: use this method to query data, such as a `SELECT` statement.
+
+        For `GET` requests, you can paginate results by specifying `api_page` and `api_page_size` as query parameters when calling the endpoint, such as `https://<region>.data.tidbcloud.com/api/v1beta/app/<App ID>/endpoint/my_endpoint/get_id?api_page=<Page Number>&api_page_size=<Page Size>`. For more information, see [Call an endpoint](#call-an-endpoint).
```

Comment:
Where is call-an-endpoint ?

Reply:
@AV25242 Here is the direct link to the "Call an end point" section: https://docs.pingcap.com/tidbcloud/data-service-manage-endpoint#call-an-endpoint

https://github.com/pingcap/docs/blob/d0d9e7e99e16afc73093c68fb78823ff422be588/tidb-cloud/data-service-manage-endpoint.md?plain=1#L179-L218